certified-connectors/Seismic Configuration/apiDefinition.swagger.json

```diff
@@ -0,0 +1,807 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Seismic Configuration",
+    "description": "Seismic configuration connector provides set of actions required for adding or updating content properties, fetching teamsites and updating email opt-outs.",
+    "contact": {
+      "name": "Seismic",
+      "email": "support@seismic.com",
+      "url": "https://seismic.com"
+    },
+    "license": {
+      "name": "seismic"
+    },
+    "version": "v2"
+  },
+  "host": "api.seismic.com",
+  "basePath": "/integration/v2",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/contentProperties/{contentPropertyId}/values": {
+      "get": {
+        "tags": [
+          "Content Properties"
+        ],
+        "summary": "Content properties - Get values of a content property",
+        "description": "Provides the list of content property values for a given content property. This will only return values for properties that have a defined list of values. Properties which have no defined domain of values and are filled in freeform by users are not provided here.",
```

Comment:
@rkushvaha we highly suggest you keep the all operations' descriptions short to fit in the box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
@Amjed-Ayoub description updated.